framework/helpers/BaseStringHelper.php

```diff
@@ -350,4 +350,19 @@ public static function base64UrlDecode($input)
     {
         return base64_decode(strtr($input, '-_', '+/'));
     }
+
+    /**
+     * Savely casts a float to string independent of the current locale.
+     *
+     * The decimal separator will always be `.`.
+     * @param float|int $number a floating point number or integer.
+     * @return string the string representation of the number.
+     * @since 2.0.13
+     */
+    public static function floatToString($number)
```

Comment:
Name doesn't reflect what it does. It's easy to think about formatting taking locale into account. How about something like `floatToDotString()`.

Reply:
well, you should read at least the short description to see what it does. Won't make the name to complex. Imo its not clear what a "dot string" is.

Reply:
How about `floatToUnifiedString()`?

Reply:
that also brings no improvement imo. The method creates a string from a float, more information is not needed from the name. The fact that this can depend on the locale and must be normalized is an implementation and documentation detail that does not need to influence the name imo. Keep it simple.

Reply:
OK.